### ISSUE TITLE
Docs: SSR docs lacking example

### ISSUE THREAD

**Issue by redbar0n** [nksaraf/vinxi#237]
The docs page:

https://vinxi.vercel.app/guide/add-to-existing-vite-app.html

Says:

> Vinxi gives you the ability to add server-side functionality (SSR) to an existing Vite app.

But the following example on the page does not show how to add SSR to an existing Vite app.

Maybe some of the TanStack Router examples could be used?

https://tanstack.com/router/latest/docs/framework/react/examples/vinxi-basic-ssr

https://tanstack.com/router/latest/docs/framework/react/examples/vinxi-basic-ssr-streaming

But maybe it would be better to show a simpler example that doesn't even have a client-side router (like TanStack Router), just SSR.

**Comment by nksaraf** [comment]
Yeah I think its a good idea. Im trying to get the docs to get a good level. Any help would be appreciated. If you feel like you can help with this ,would love that
